ci-operator/step-registry/aws/deprovision/stacks/aws-deprovision-stacks-ref.yaml

```diff
@@ -1,14 +1,14 @@
 ref:
-  as: aws-deprovision-vpc
+  as: aws-deprovision-stacks
   from_image:
     namespace: ocp
     name: "4.5"
     tag: upi-installer
   grace_period: 10m
-  commands: aws-deprovision-vpc-commands.sh
+  commands: aws-deprovision-stacks-commands.sh
   resources:
     requests:
       cpu: 10m
       memory: 100Mi
   documentation: |-
-    Delete VPC.
+    Delete AWS CloudFormation stacks.
```

Comment:
How about 'Delete AWS CloudFormation stacks which were created in "aws-provision-bastionhost", ...'?